### PR TITLE
Initial configurations for the radxa zerow

### PR DESCRIPTION
**Pull request by GASB0**
I haven't tested the generated image yet, but what's available so far should be enough to start building.

The configurations here are mostly based on the configurations for the CM3 board from the
git@github.com:flatmax/buildroot.rockchip.git
repository.

### CODE REVIEW

**Comment by GASB0** [comment]
Fixing uboot building. Now the uboot version built for this repo is as specified in:
https://wiki.radxa.com/Zero/dev/u-boot
Next commit should be incorporating the Firmware in Package and then generate the image.

**Comment by GASB0** [comment]
Finally getting the custom Linux image to boot on the Radxa Zero board in my testings.

File: external.desc [hunk 1]
```diff
@@ -0,0 +1 @@
+name: CUSTOM_RADXA_ZEROW
```
Comment (flatmax) [inline external.desc]: Change this to CUSTOM_AUDIO_INJECTOR

Comment (flatmax) [reply external.desc]: I expect you';ll have to change :
BR2_EXTERNAL_CUSTOM_RADXA_ZEROW_PATH
To :
BR2_EXTERNAL_CUSTOM_AUDIO_INJECTOR_PATH

File: board/S905Y2/kernel.fragments
Comment (flatmax) [inline board/S905Y2/kernel.fragments:1]: Why do you have these here - aren't they defined by the default defconfig for linux already ?

File: board/S905Y2/post-build.sh
Comment (flatmax) [inline board/S905Y2/post-build.sh:1]: this is out of date - possibly remove it ?

File: configs/zerow_defconfig [hunk 2]
```diff
@@ -0,0 +1,63 @@
+BR2_TARGET_GENERIC_HOSTNAME="zero_w"
```
Comment (flatmax) [inline configs/zerow_defconfig]: change to audio_injector

File: configs/zerow_defconfig [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+BR2_TARGET_GENERIC_ISSUE="Welcome to the zero_w board"
```
Comment (flatmax) [inline configs/zerow_defconfig]: Change to :
Welcome to the Audio Injector OS

File: configs/zerow_defconfig [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+BR2_PACKAGE_HOST_MTOOLS=y
+BR2_PACKAGE_HOST_UBOOT_TOOLS=y
+BR2_PACKAGE_HOST_GENIMAGE=y
+
+# u-boot
+BR2_TARGET_UBOOT=y
+BR2_TARGET_UBOOT_BUILD_SYSTEM_KCONFIG=y
+BR2_TARGET_UBOOT_USE_DEFCONFIG=y
+BR2_TARGET_UBOOT_CUSTOM_GIT=y
+BR2_TARGET_UBOOT_CUSTOM_REPO_URL="https://github.com/radxa/u-boot.git"
```
Comment (flatmax) [inline configs/zerow_defconfig:25]: I notice you didn't use the stock uboot - perhaps we can talk about that on signal. OK for now,

File: configs/zerow_defconfig [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+BR2_TARGET_UBOOT_CUSTOM_REPO_VERSION="radxa-zero-v2021.07"
+BR2_TARGET_UBOOT_BOARD_DEFCONFIG="radxa-zero2"
+BR2_TARGET_UBOOT_CONFIG_FRAGMENT_FILES="$(BR2_EXTERNAL)/board/S905Y2/uboot.fragment"
+
+# linux
+BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_LATEST_VERSION=y
```
Comment (flatmax) [inline configs/zerow_defconfig:32]: fantastic !

**Review by flatmax — CHANGES_REQUESTED** [review]
Fantastic work - really good start. Hardly anything to change. Well done.

File: configs/zerow_defconfig (hunk 4 above)
Comment (GASB0) [reply configs/zerow_defconfig:25]: I was looking into the radxa wiki:
https://wiki.radxa.com/Zero/dev/u-boot
And that's the repo they used in their example. So I took it from there.

**Review by flatmax — APPROVED** [review]

**Pull request merged**